.github/workflows/CI.yml

```diff
@@ -77,13 +77,3 @@ jobs:
         with:
           fail_ci_if_error: true
           directory: app/
-
-      - name: Run linter
-        run: |
-          pipenv run pylint tests/ referee/ core/ sampl/ ../ever_given/ever_given/
-        env:
-          DJANGO_SETTINGS_MODULE: sampl.settings_dev
-      - name: Run formatting checks
-        run: |
-          pipenv run black --check --diff .
-          pipenv run isort --check --diff core/ referee/ sampl/ tests/  conftest.py
```

Comment:
curious - why not isort .?

Reply:
We will run isort on pre-commit.ci and if it needs to, it can change the order of the imports and make a new commit instead of us having to run it locally then commit the result.